### PR TITLE
Remove crh created practices endpoint

### PR DESCRIPTION
**Pull request by PhilipDeFraties**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-4209

## Description - what does this code do?
Removes action, route, model method, and js related to `crh/:id/crh-created-practices` endpoint.

The endpoint appears to have been intended to be a source of data for back-filling the "created practices" section on a `clinical-resource-hub`s show page. However this info is already provided in the initial request: https://github.com/department-of-veterans-affairs/diffusion-marketplace/blob/c4ac9916b02cc14202653a613e4da2864b707ed9/app/controllers/clinical_resource_hubs_controller.rb#L9

Also the fetch request in the `app/assets/javascripts/clinical_resource_hubs/crh_show.es6` file was set up to insert the returned data into a css table class that does not exist..

## Testing done - how did you test it/steps on how can another person can test it 
in rails console run the following commands:

```
include FactoryBot::Syntax::Methods
create(:practice_origin_facility, practice: Practice.find(1), clinical_resource_hub: ClinicalResourceHub.find(1))
create(:practice_origin_facility, practice: Practice.find(2), clinical_resource_hub: ClinicalResourceHub.find(1))
create(:practice_origin_facility, practice: Practice.find(3), clinical_resource_hub: ClinicalResourceHub.find(1))
create(:diffusion_history, :with_va_facility, :with_status_complete, practice: Practice.find(1))
```

nav to `http://localhost:3200/crh/1` and verify 3 practices show up in the "Innovations created here" section, verify the filter and sort functions as expected.

You can verify the categories of a given practice in the console with:
`Practice.find(3).categories.map(&:name)`

## Screenshots, Gifs, Videos from application (if applicable)
Screenshot 2023-09-27 at 12 00 11 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/controllers/clinical_resource_hubs_controller.rb [hunk 1]
```diff
@@ -20,31 +20,6 @@ def show
     get_categories_by_practices(@practices_adopted_by_crh, @crh_practices_adopted_categories)
   end
 
-  # GET /crh/:id/created_crh_practices
-  def created_crh_practices
```
Comment (camillevilla) [inline app/controllers/clinical_resource_hubs_controller.rb:24]: Let's remove this from the `before_action` in line 4 as well

File: spec/factories/diffusion_histories.rb [hunk 2]
```diff
@@ -10,6 +10,10 @@
       association :clinical_resource_hub
     end
 
+    # there are places in the application that expect a diffusion_history to have an associated
+    # diffusion_history_status, if you create a diffusion_history in rails console without one
+    # of the :with_status traits it will cause a breakage, ie /diffusion-map
```
Comment (camillevilla) [inline spec/factories/diffusion_histories.rb:15]: thanks for the helpful note here! We can add more error handling in diffusion map to our tech debt tickets

**Review by camillevilla — APPROVED** [review]
@PhilipDeFraties This is looking good! I was able to load the CRH pages and practices successfully with the new code. Having factories and more test coverage for this part of the codebase is a great asset. I have 1 small cleanup request and this is good to go.

File: app/controllers/clinical_resource_hubs_controller.rb (hunk 1 above)
Comment (PhilipDeFraties) [reply app/controllers/clinical_resource_hubs_controller.rb:24]: good eye, thank you for catching that!

**Pull request merged**